<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HTTP</class>
 <widget class="QWidget" name="HTTP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HTTP测试_ScmTble</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QComboBox" name="method">
       <property name="styleSheet">
        <string notr="true">font: 75 20pt &quot;Consolas&quot;;</string>
       </property>
       <item>
        <property name="text">
         <string>GET</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>POST</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="url">
       <property name="styleSheet">
        <string notr="true">font: 75 20pt &quot;Consolas&quot;;</string>
       </property>
       <property name="placeholderText">
        <string>URL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="yes">
       <property name="styleSheet">
        <string notr="true">font: 75 20pt &quot;Consolas&quot;;</string>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">font: 75 15pt &quot;Consolas&quot;;</string>
     </property>
     <property name="title">
      <string>参数设置</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="clearre_button">
            <property name="text">
             <string>清除响应</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearurl_button">
            <property name="text">
             <string>清除URL</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearall_button">
            <property name="text">
             <string>全部清空</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="addwt_button">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="increasewt_button">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="search_button">
            <property name="text">
             <string>查找</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTabWidget" name="Tab">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="params">
             <attribute name="title">
              <string>params</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <widget class="QPushButton" name="add_button">
                    <property name="text">
                     <string>+</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="increase_button">
                    <property name="text">
                     <string>-</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QTableWidget" name="params_text">
                  <property name="styleSheet">
                   <string notr="true">font: 75 15pt &quot;Consolas&quot;;</string>
                  </property>
                  <attribute name="horizontalHeaderStretchLastSection">
                   <bool>true</bool>
                  </attribute>
                  <column>
                   <property name="text">
                    <string>参数</string>
                   </property>
                   <property name="font">
                    <font>
                     <family>Consolas</family>
                    </font>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>值</string>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="heardes">
             <attribute name="title">
              <string>headers</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <widget class="QPushButton" name="headadd_button">
                    <property name="text">
                     <string>+</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="headincrease_button">
                    <property name="text">
                     <string>-</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QTableWidget" name="headers_text">
                  <property name="styleSheet">
                   <string notr="true">font: 75 15pt &quot;Consolas&quot;;</string>
                  </property>
                  <attribute name="horizontalHeaderStretchLastSection">
                   <bool>true</bool>
                  </attribute>
                  <column>
                   <property name="text">
                    <string>请求头</string>
                   </property>
                   <property name="font">
                    <font>
                     <family>Consolas</family>
                    </font>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>值</string>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="what_text">
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>变量</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>值</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">font: 75 15pt &quot;Consolas&quot;;</string>
     </property>
     <property name="title">
      <string>响应内容</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QTextBrowser" name="retext">
        <property name="documentTitle">
         <string>HTTP</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
